pycds/orm/tables.py

```diff
@@ -304,6 +321,24 @@ class Variable(Base):
             "net_var_name ~ '^[a-zA-Z_][a-zA-Z0-9_$]*$'",
             name="ck_net_var_name_valid_identifier",
         ),
+        CheckConstraint(no_newline_ck_check("unit"), name=no_newline_ck_name("unit")),
+        CheckConstraint(
+            no_newline_ck_check("short_name"), name=no_newline_ck_name("short_name")
+        ),
+        CheckConstraint(
+            no_newline_ck_check("display_name"), name=no_newline_ck_name("display_name")
+        ),
+        CheckConstraint(
+            no_newline_ck_check("cell_method"), name=no_newline_ck_name("cell_method")
+        ),
+        CheckConstraint(
+            no_newline_ck_check("standard_name"),
+            name=no_newline_ck_name("standard_name"),
+        ),
+        CheckConstraint(
+            no_newline_ck_check("long_description"),
+            name=no_newline_ck_name("long_description"),
+        ),
```

Comment:
Two thoughts:

You can express this as

```python
no_newline_ck_columns = ("unit", ...)

class Variable:
    ...
    __table_args__ = (...) + tuple(
        CheckConstraint(no_newline_ck_check(column), name=no_newline_ck_name(column) 
        for column in no_newline_ck_columns
    )
```

and you can then import `no_newline_ck_columns` into the migration script, DRYing things up and avoiding errors.

Reply:
As long as `CheckConstraint` takes rendered column names rather than the SQLAlchemy object column names as the argument. Which I think is true, and your tests would fail if not. But worth asking.

Reply:
I considered this, I chose in this instance to leave it fully textual as it lets people see at a glance which columns the constraint is applied to directly as part of the ORM definition without having to jump to another location (ie a list of columns somewhere else). In other parts of the migration this didn't seem as critical due to them just being operations. Thoughts?

Reply:
I've changed it based on your suggestion but located the variable that holds the list of column's close to the usage. See what you think (and if its safe for me to use an underscore variable here)